ui/components/multichain/balance-overview/balance-overview.js

```diff
@@ -157,7 +160,7 @@ export const BalanceOverview = () => {
                 location: 'Home',
                 text: 'Portfolio',
                 chain_id: chainId,
-                token_symbol: 'ETH',
+                token_symbol: CURRENCY_SYMBOLS.ETH,
```

Comment:
Question: if it is not in `(build-main,build-beta,build-flask)` mode, `CURRENCY_SYMBOLS` would be undefined? maybe we should do 
```
token_symbol: CURRENCY_SYMBOLS.ETH | `ETH`",
```

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/21093, good catch!